### PR TITLE
 teams/tidb: a vote for yangkeao as TiDB commiter

### PR DESCRIPTION
**Pull request by bb7133**
As stated.

### CODE REVIEW

File: teams/tidb/membership.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "maintainers": [
         "3pointer",
         "XuHuaiyu",
+        "YuJuncen",
```
Comment (tiancaiamao) [inline teams/tidb/membership.json:7]: Case sensitive sorting?

**Review by tiancaiamao — APPROVED** [review]

**Review by CbcWestwolf — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Review by joccau — APPROVED** [review]
LGTM

File: teams/tidb/membership.json (hunk 1 above)
Comment (bb7133) [reply teams/tidb/membership.json:7]: Yes, following the current collation: https://github.com/pingcap/community/blob/master/teams/tidb/membership.json

**Review by breezewish — APPROVED** [review]
Congrats!

**Pull request merged**